<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="NANDAKUMARSIR-D" name="Regression" tests="25" failures="5" timestamp="20 Jun 2017 17:36:50 GMT" time="869.425" errors="0">
  <testcase name="checkPrimeNumber" time="0.009" classname="javapracticeone.PracticeMethods"/>
  <testcase name="printEvenNumber" time="0.002" classname="javapracticeone.PracticeMethods"/>
  <testcase name="usingLoops" time="6.005" classname="javapracticeone.PracticeMethods"/>
  <testcase name="usingDropDown" time="45.692" classname="javapracticeone.PracticeMethods">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at javapracticeone.PracticeMethods.usingDropDown(PracticeMethods.java:171)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 7 stack frames]]>
    </failure>
  </testcase> <!-- usingDropDown -->
  <testcase name="fibonacciSeries" time="0.001" classname="javapracticeone.PracticeMethods"/>
  <testcase name="dragAndDrop" time="61.573" classname="javapracticeone.PracticeMethodsOne"/>
  <testcase name="usingRightClick" time="50.928" classname="javapracticeone.PracticeMethodsOne">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at javapracticeone.PracticeMethodsOne.usingRightClick(PracticeMethodsOne.java:111)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- usingRightClick -->
  <testcase name="useTwitterUsingObjectRepo" time="93.943" classname="javapracticeone.PracticeMethods">
    <failure type="org.testng.TestException" message="
Method PracticeMethods.useTwitterUsingObjectRepo()[pri:6, instance:javapracticeone.PracticeMethods@df4efb] should have thrown an exception of type class org.openqa.selenium.WebDriverException">
      <![CDATA[org.testng.TestException: 
Method PracticeMethods.useTwitterUsingObjectRepo()[pri:6, instance:javapracticeone.PracticeMethods@df4efb] should have thrown an exception of type class org.openqa.selenium.WebDriverException
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 7 stack frames]]>
    </failure>
  </testcase> <!-- useTwitterUsingObjectRepo -->
  <testcase name="runInGrid" time="57.288" classname="javapracticeone.PracticeMethodsOne"/>
  <testcase name="printLinks" time="114.517" classname="javapracticeone.PracticeMethods"/>
  <testcase name="countDropdown" time="20.492" classname="javapracticeone.PracticeMethods"/>
  <testcase name="printAllLinks" time="22.1" classname="javapracticeone.PracticeMethods"/>
  <testcase name="useAllMethods" time="30.635" classname="javapracticeone.PracticeMethods"/>
  <testcase name="handlingAlerts" time="22.435" classname="javapracticeone.PracticeMethods"/>
  <testcase name="handlingFrame" time="21.706" classname="javapracticeone.PracticeMethods"/>
  <testcase name="handlingPrompts" time="28.988" classname="javapracticeone.PracticeMethods">
    <failure type="org.testng.TestException" message="
Method PracticeMethods.handlingPrompts()[pri:13, instance:javapracticeone.PracticeMethods@df4efb] should have thrown an exception of type class org.openqa.selenium.UnhandledAlertException">
      <![CDATA[org.testng.TestException: 
Method PracticeMethods.handlingPrompts()[pri:13, instance:javapracticeone.PracticeMethods@df4efb] should have thrown an exception of type class org.openqa.selenium.UnhandledAlertException
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 7 stack frames]]>
    </failure>
  </testcase> <!-- handlingPrompts -->
  <testcase name="handlingWindows" time="25.442" classname="javapracticeone.PracticeMethods"/>
  <testcase name="countFrames" time="43.394" classname="javapracticeone.PracticeMethods">
    <failure type="org.openqa.selenium.remote.UnreachableBrowserException" message="Error communicating with the remote browser. It may have died.
Build info: version: &amp;apos;2.53.1&amp;apos;, revision: &amp;apos;a36b8b1&amp;apos;, time: &amp;apos;2016-06-30 17:37:03&amp;apos;
System info: host: &amp;apos;NANDAKUMARSIR-D&amp;apos;, ip: &amp;apos;192.168.56.1&amp;apos;, os.name: &amp;apos;Windows 7&amp;apos;, os.arch: &amp;apos;x86&amp;apos;, os.version: &amp;apos;6.1&amp;apos;, java.version: &amp;apos;1.8.0&amp;apos;
Driver info: driver.version: FirefoxDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=47.0.2, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: 57338131-d6d7-4c50-ac42-5a444aa29d9b">
      <![CDATA[org.openqa.selenium.remote.UnreachableBrowserException: Error communicating with the remote browser. It may have died.
Build info: version: '2.53.1', revision: 'a36b8b1', time: '2016-06-30 17:37:03'
System info: host: 'NANDAKUMARSIR-D', ip: '192.168.56.1', os.name: 'Windows 7', os.arch: 'x86', os.version: '6.1', java.version: '1.8.0'
Driver info: driver.version: FirefoxDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=47.0.2, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: 57338131-d6d7-4c50-ac42-5a444aa29d9b
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:665)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:701)
	at org.openqa.selenium.firefox.FirefoxDriver.getScreenshotAs(FirefoxDriver.java:341)
	at javapracticeone.PracticeMethods.countFrames(PracticeMethods.java:629)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.apache.http.conn.HttpHostConnectException: Connect to 127.0.0.1:7055 [/127.0.0.1] failed: Connection refused: connect
	at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:151)
	at org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:353)
	at org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:380)
	at org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:236)
	at org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:184)
	at org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:88)
	at org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:110)
	at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:184)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:71)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:55)
	at org.openqa.selenium.remote.internal.ApacheHttpClient.fallBackExecute(ApacheHttpClient.java:144)
	at org.openqa.selenium.remote.internal.ApacheHttpClient.execute(ApacheHttpClient.java:90)
	at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:142)
	at org.openqa.selenium.firefox.internal.NewProfileExtensionConnection.execute(NewProfileExtensionConnection.java:160)
	at org.openqa.selenium.firefox.FirefoxDriver$LazyCommandExecutor.execute(FirefoxDriver.java:380)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:644)
	... 16 more
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.doConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connectToAddress(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connect(Unknown Source)
	at java.net.PlainSocketImpl.connect(Unknown Source)
	at java.net.SocksSocketImpl.connect(Unknown Source)
	at java.net.Socket.connect(Unknown Source)
	at org.apache.http.conn.socket.PlainConnectionSocketFactory.connectSocket(PlainConnectionSocketFactory.java:74)
	at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:134)
	... 31 more
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- countFrames -->
  <testcase name="draggableAndDroppable" time="329.271" classname="javapracticeone.PracticeMethodsOne"/>
  <testcase name="closeFrames" time="60.394" classname="javapracticeone.PracticeMethods"/>
  <testcase name="readContent" time="28.436" classname="javapracticeone.PracticeMethods"/>
  <testcase name="starPattern" time="0.001" classname="javapracticeone.PracticeMethods"/>
  <testcase name="usingWrapperMethods" time="129.82" classname="javapracticeone.PracticeMethods"/>
  <testcase name="dataUsingDataProvider" time="0.0" classname="javapracticeone.PracticeMethods">
    <skipped/>
  </testcase> <!-- dataUsingDataProvider -->
  <testcase name="selectableRunInGrid" time="369.328" classname="javapracticeone.PracticeMethodsOne"/>
</testsuite> <!-- Regression -->
